presto-password-authenticators/src/main/java/io/prestosql/plugin/password/ldap/LdapConfig.java

```diff
@@ -33,9 +33,10 @@
     private String bindPassword;
     private boolean ignoreReferrals;
     private Duration ldapCacheTtl = new Duration(1, TimeUnit.HOURS);
+    private boolean allowInsecure;
```

Comment:
Move this and the getter/setter to be directly after the URL, since they are related.